compute/vm_test.go

```diff
@@ -489,3 +491,41 @@ func Example_createVMWithUserAssignedIdentity() {
 	// added second user-assigned identity to VM
 	// removed first user-assigned identity from VM
 }
+
+func Example_list() {
+	// list the VMs we've created in our resource group by page.
+	// uses the default page size returned by the service.
+	vmClient := getVMClient()
+	for page, err := vmClient.List(context.Background(), config.GroupName()); page.NotDone(); err = page.Next() {
+		if err != nil {
+			util.LogAndPanic(err)
```

Comment:
Is log and panic common in these samples?

Reply:
Yep, I copied this from one of the other samples in this file.